app/Services/Doctor/VaccinationSlotService.cs

```diff
@@ -0,0 +1,37 @@
+﻿using backend.Database;
+using backend.Dto.Requests.Doctor;
+using backend.Dto.Responses.Doctor;
+using backend.Exceptions;
+using backend.Models.Accounts;
+using backend.Models.Visits;
+
+namespace backend.Services.Doctor
+{
+    public class VaccinationSlotService
+    {
+        private readonly DataContext dataContext;
+        public VaccinationSlotService(DataContext dataContext)
+        {
+            this.dataContext = dataContext;
+        }
+
+        public async Task<NewVaccinationSlotResponse> AddNewSlot(NewVaccinationSlotRequest request, DoctorModel doctor)
+        {
+            DateTime date;
+
+            try
```

Comment:
Mogę dodać kilka slotów na tą samą datę i godzinę. Może jakieś ograniczenie, że slot to 15min i jak dodam o 10:00 to następny może być o 10:15 dopiero a wcześniejszy minimalnie o 9:45
![image](https://user-images.githubusercontent.com/19613420/159117273-3532a924-1bf6-4a05-b417-70e54ec05145.png)

Reply:
Może omówmy to jako nowy task przy okazji następnych sprintów? Podejrzewam, że musielibyśmy wprowadzić dodatkową odpowiedź w API, żeby zaznaczyć, że błąd wynika ze zbyt bliskich terminów, co musielibyśmy ustalić z pozostałymi grupami oraz obsłużyć na frontendzie.

Reply:
Zrobione 👍 
Dodaliśmy sprawdzanie, czy slot nie nachodzi o 15 min.
W przypadku gdyby nachodził to zwracamy ValidationError (to możemy w przyszłości zmienić na coś bardziej konkretnego).